documentation/source/users/cantherm/input.rst

```diff
@@ -33,16 +34,27 @@ Function                    Description
 Model Chemistry
 ===============
 
-The first item in the input file should be a ``modelChemistry()`` function,
-which accepts a string describing the model chemistry.
+The first item in the input file should be a ``modelChemistry`` assignment
+with a string describing the model chemistry.
 
 CanTherm uses this information to adjust the computed energies to the usual gas-phase reference
 states by applying atom, bond and spin-orbit coupling energy corrections. This is particularly
 important for ``thermo()`` calculations (see below). Note that the user must specify under the
-``species()`` function the type and number of atoms and bonds for CanTherm to apply these corrections.
+``species()`` function the type and number of bonds for CanTherm to apply these corrections.
 The example below specifies CBS-QB3 as the model chemistry::
 
-    modelChemistry("CBS-QB3")
+    modelChemistry = "CBS-QB3"
+
+Alternatively, the atomic energies at the ``modelChemistry`` level of theory can be directly
```

Comment:
We should tell the users that no bond corrections will be applied in this case, and perhaps allow them to add their own?

Reply:
Bond corrections would still be applied if `modelChemistry` matches one of the options in `applyEnergyCorrections`, although of course it is unlikely that someone would specify `atomEnergies` themselves if they are already available within Cantherm. I could add an option to add bond corrections, but deriving them is typically associated with a lot more work than calculating atomic energies, so I doubt that option would be used. Ideally, if new bond corrections become available, they should be added to Cantherm. What do you think?

Reply:
I think it's enough to mention in the documentation something like:
If a `modelChemistry` other than the ones specified in the table is given, then the user should add the respective `atomEnergies` to get meaningful results, and BAC would not be applied.